### DISCUSSION TITLE
URL naming

### DISCUSSION THREAD

**Discussion by Jimmy291** [greenelab/lab-website-template#265] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://jimmy291.github.io/swift.github.io/

### Version of Lab Website Template you are using

1.2.2

### Description

I would like to ask how do I change the URL? 

Now the URL is https://userID.github.io/your-lab.github.io, how can I change it to https://your-lab.github.io

**Comment by vincerubinetti** [comment, marked as answer]
This is shown in the docs here:
https://greene-lab.gitbook.io/lab-website-template-docs/getting-started/set-up-your-url

If your username is `jimmy291`, you need to name your repo `jimmy291.github.io` and your website will appear at `https://jimmy291.github.io`. If you wanted your website to be at `https://swift.github.io`, you would need to change your username to `swift` (and I'm certain that username is already taken). This is a [limitation of GitHub itself](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites), not the template. GitHub is providing you a domain name for free, and says that it must match your user/org name. I would recommend buying a custom domain name like `swiftlab.com`, [as described in the docs](https://greene-lab.gitbook.io/lab-website-template-docs/getting-started/set-up-your-url).
